core/src/test/java/org/apache/iceberg/TestRewriteManifests.java

```diff
@@ -743,4 +887,16 @@ private void validateSummary(Snapshot snapshot, int replaced, int kept, int crea
         "Entry count should match",
         entryCount, Integer.parseInt(summary.get("entries-processed")));
   }
+
+  private void matchNumberOfManifestFileWithSpecId(List<ManifestFile> manifestFiles, int toBeMatchedPartitionSpecId,
```

Comment:
In theory, we can replace this method with these lines:

```
int manifestsWithInitialSpecId = manifests.stream().filter(m -> m.partitionSpecId() == xxx).count();
Assert.assertEquals("...", 3, manifestsWithInitialSpecId);
```

Reply:
Or just use that code in this method.

Reply:
done.